.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+defaults:
+  run:
+    shell: bash
```

Comment:
I haven't seen this before - I thought bash was the default?

Reply:
I started with wit-bindgen's workflow. Running locally doesn't appear to be needed.